tests/models/codegen/test_tokenization_codegen.py

```diff
@@ -263,3 +263,12 @@ def test_truncation(self):
     # tokenizer has no padding token
     def test_padding_different_model_input_name(self):
         pass
+
+    # @slow
```

Comment:
```suggestion
    @slow
```

Reply:
Sorry I forgot to uncomment it. I've fixed it!